[{"count": 0, "tags": [], "text": "Created attachment 35480\nthe jmeter log\n\nCan't open a script in apache-jmeter-r1814153 with Java 9 on Windows - getting:\n\n\n2017-11-03 14:49:47,161 INFO o.a.j.g.a.Load: Loading file: C:\\projects\\JMeter\\dmcs_171_modified.jmx\n2017-11-03 14:49:47,162 INFO o.a.j.s.FileServer: Set new base='C:\\projects\\JMeter'\n2017-11-03 14:49:47,339 ERROR o.a.j.JMeter: Uncaught exception: \njava.lang.ExceptionInInitializerError: null\n\tat com.thoughtworks.xstream.XStream.setupConverters(XStream.java:989) ~[xstream-1.4.10.jar:1.4.10]\n\tat com.thoughtworks.xstream.XStream.<init>(XStream.java:592) ~[xstream-1.4.10.jar:1.4.10]\n\tat com.thoughtworks.xstream.XStream.<init>(XStream.java:514) ~[xstream-1.4.10.jar:1.4.10]\n\tat com.thoughtworks.xstream.XStream.<init>(XStream.java:483) ~[xstream-1.4.10.jar:1.4.10]\n\tat com.thoughtworks.xstream.XStream.<init>(XStream.java:429) ~[xstream-1.4.10.jar:1.4.10]\n\tat com.thoughtworks.xstream.XStream.<init>(XStream.java:382) ~[xstream-1.4.10.jar:1.4.10]\n\tat org.apache.jmeter.save.SaveService$XStreamWrapper.<init>(SaveService.java:82) ~[ApacheJMeter_core.jar:r1814153]\n\tat org.apache.jmeter.save.SaveService$XStreamWrapper.<init>(SaveService.java:80) ~[ApacheJMeter_core.jar:r1814153]\n\tat org.apache.jmeter.save.SaveService.<clinit>(SaveService.java:113) ~[ApacheJMeter_core.jar:r1814153]\n\tat org.apache.jmeter.gui.action.Load.loadProjectFile(Load.java:130) ~[ApacheJMeter_core.jar:r1814153]\n\tat org.apache.jmeter.gui.action.Load.loadProjectFile(Load.java:102) ~[ApacheJMeter_core.jar:r1814153]\n\tat org.apache.jmeter.gui.action.Load.doActionAfterCheck(Load.java:89) ~[ApacheJMeter_core.jar:r1814153]\n\tat org.apache.jmeter.gui.action.AbstractActionWithNoRunningTest.doAction(AbstractActionWithNoRunningTest.java:45) ~[ApacheJMeter_core.jar:r1814153]\n\tat org.apache.jmeter.gui.action.ActionRouter.performAction(ActionRouter.java:88) ~[ApacheJMeter_core.jar:r1814153]\n\tat org.apache.jmeter.gui.action.ActionRouter.lambda$actionPerformed$0(ActionRouter.java:70) ~[ApacheJMeter_core.jar:r1814153]\n\tat java.awt.event.InvocationEvent.dispatch(Unknown Source) ~[?:?]\n\tat java.awt.EventQueue.dispatchEventImpl(Unknown Source) ~[?:?]\n\tat java.awt.EventQueue.access$500(Unknown Source) ~[?:?]\n\tat java.awt.EventQueue$3.run(Unknown Source) ~[?:?]\n\tat java.awt.EventQueue$3.run(Unknown Source) ~[?:?]\n\tat java.security.AccessController.doPrivileged(Native Method) ~[?:?]\n\tat java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(Unknown Source) ~[?:?]\n\tat java.awt.EventQueue.dispatchEvent(Unknown Source) ~[?:?]\n\tat java.awt.EventDispatchThread.pumpOneEventForFilters(Unknown Source) [?:?]\n\tat java.awt.EventDispatchThread.pumpEventsForFilter(Unknown Source) [?:?]\n\tat java.awt.EventDispatchThread.pumpEventsForHierarchy(Unknown Source) [?:?]\n\tat java.awt.EventDispatchThread.pumpEvents(Unknown Source) [?:?]\n\tat java.awt.EventDispatchThread.pumpEvents(Unknown Source) [?:?]\n\tat java.awt.EventDispatchThread.run(Unknown Source) [?:?]\nCaused by: java.lang.reflect.InaccessibleObjectException: Unable to make field private final java.util.Comparator java.util.TreeMap.comparator accessible: module java.base does not \"opens java.util\" to unnamed module @554e218\n\tat java.lang.reflect.AccessibleObject.checkCanSetAccessible(Unknown Source) ~[?:?]\n\tat java.lang.reflect.AccessibleObject.checkCanSetAccessible(Unknown Source) ~[?:?]\n\tat java.lang.reflect.Field.checkCanSetAccessible(Unknown Source) ~[?:?]\n\tat java.lang.reflect.Field.setAccessible(Unknown Source) ~[?:?]\n\tat com.thoughtworks.xstream.core.util.Fields.locate(Fields.java:40) ~[xstream-1.4.10.jar:1.4.10]\n\tat com.thoughtworks.xstream.converters.collections.TreeMapConverter.<clinit>(TreeMapConverter.java:50) ~[xstream-1.4.10.jar:1.4.10]\n\t... 29 more\n\n\njava -version\njava version \"9-ea\"\nJava(TM) SE Runtime Environment (build 9-ea+163)\nJava HotSpot(TM) 64-Bit Server VM (build 9-ea+163, mixed mode)", "is_private": false, "bug_id": 61721, "id": 201887, "time": "2017-11-03T20:33:05Z", "creator": "apodelko@yahoo.com", "creation_time": "2017-11-03T20:33:05Z", "attachment_id": 35480}, {"count": 1, "tags": [], "creator": "apodelko@yahoo.com", "text": "Created attachment 35481\none of the scripts I tried to open", "id": 201888, "time": "2017-11-03T20:36:46Z", "bug_id": 61721, "creation_time": "2017-11-03T20:36:46Z", "is_private": false, "attachment_id": 35481}, {"count": 2, "tags": [], "bug_id": 61721, "is_private": false, "id": 201889, "attachment_id": null, "creator": "p.mouawad@ubik-ingenierie.com", "creation_time": "2017-11-03T20:49:27Z", "time": "2017-11-03T20:49:27Z", "text": "Hello,\nIt seems you may be using an Early access version of Jdk.\n\nCan you try using last version of JDK 9 ?\n\nThanks"}, {"count": 3, "tags": [], "bug_id": 61721, "text": "I also notice an issue I don't reproduce with more recent version of Java:\n\n2017-11-03 14:49:12,032 WARN o.a.j.JMeter: Could not set LAF to: Darcula\njava.lang.ClassNotFoundException: Darcula\n\tat java.net.URLClassLoader.findClass(Unknown Source) ~[?:?]\n\tat java.lang.ClassLoader.loadClass(Unknown Source) ~[?:?]\n\tat java.lang.ClassLoader.loadClass(Unknown Source) ~[?:?]\n\tat java.lang.Class.forName0(Native Method) ~[?:?]\n\tat java.lang.Class.forName(Unknown Source) ~[?:?]\n\tat javax.swing.SwingUtilities.loadSystemClass(Unknown Source) ~[?:?]\n\tat javax.swing.UIManager.setLookAndFeel(Unknown Source) ~[?:?]\n\tat org.apache.jmeter.JMeter.startGui(JMeter.java:358) [ApacheJMeter_core.jar:r1814153]\n\tat org.apache.jmeter.JMeter.start(JMeter.java:519) [ApacheJMeter_core.jar:r1814153]\n\tat jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]\n\tat jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source) ~[?:?]\n\tat jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source) ~[?:?]\n\tat java.lang.reflect.Method.invoke(Unknown Source) ~[?:?]\n\tat org.apache.jmeter.NewDriver.main(NewDriver.java:248) [ApacheJMeter.jar:r1814153]\n\n\nAnyway I don't reproduce your issue on Java 9+181", "id": 201890, "time": "2017-11-03T21:05:03Z", "creator": "p.mouawad@ubik-ingenierie.com", "creation_time": "2017-11-03T21:05:03Z", "is_private": false, "attachment_id": null}, {"count": 4, "tags": [], "bug_id": 61721, "attachment_id": null, "text": "Yes, with the latest version:\n\n>java -version\njava version \"9.0.1\"\nJava(TM) SE Runtime Environment (build 9.0.1+11)\nJava HotSpot(TM) 64-Bit Server VM (build 9.0.1+11, mixed mode)\n\nit looks like I can open my script. However it shows some warnings when starts:\n\nWARNING: An illegal reflective access operation has occurred\nWARNING: Illegal reflective access by org.codehaus.groovy.reflection.CachedClass (file:/C:/apache-jmeter-r1814153/lib/groovy-all-2.4.12.jar) to method java.lang.Object.finalize()\nWARNING: Please consider reporting this to the maintainers of org.codehaus.groovy.reflection.CachedClass\nWARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations\nWARNING: All illegal access operations will be denied in a future release", "id": 201918, "time": "2017-11-05T02:37:21Z", "creator": "apodelko@yahoo.com", "creation_time": "2017-11-05T02:37:21Z", "is_private": false}, {"attachment_id": null, "tags": [], "bug_id": 61721, "text": "Hello Alexander,\nThanks for feedback.\nThe issues you mention are identified and are in groovy current version.\n\nIs the darcula theme working also now for you ?\n\n\nThanks", "count": 5, "id": 201925, "time": "2017-11-05T10:30:32Z", "creator": "p.mouawad@ubik-ingenierie.com", "creation_time": "2017-11-05T10:30:32Z", "is_private": false}, {"count": 6, "tags": [], "text": "\n\n*** This bug has been marked as a duplicate of bug 61529 ***", "is_private": false, "id": 202007, "creator": "p.mouawad@ubik-ingenierie.com", "time": "2017-11-08T21:13:48Z", "bug_id": 61721, "creation_time": "2017-11-08T21:13:48Z", "attachment_id": null}]